lib/subprocess.js

```diff
@@ -170,31 +153,6 @@ class SubProcess extends EventEmitter {
 
         // emit the actual output for whomever's listening
         this.emit('output', stdout, stderr);
-
-        // we also want to emit lines, but it's more complex since output
```

Comment:
so we just don't need this logic anymore? how come?

Reply:
it's still there, it's just based on a different foundation. Tests remain unchanged and the logic is preserved